### ISSUE TITLE
When user is not logged in to any cluster, the extension still show the latest kube context and namespace field shows error

### ISSUE THREAD

**Issue by lgrossma** [redhat-developer/intellij-openshift-connector#631]
When user is logged out of a cluster, the extension still shows the latest kube context. This is not much of a problem but when the namespace field is viewed, it shows error:

Could not get namespaces: Failure executing: GET at: https://api.ocp1.adapters-crs.ccitredhat.com:6443/api/v1/namespaces. Message: Forbidden! User developer/api-ocp1-adapters-crs-ccitredhat-com:6443 doesn't have permission. namespaces is forbidden: User "system:anonymous" cannot list resource "namespaces" in API group "" at the cluster scope.

which does not feel really good.

**Comment by sbouchet** [comment]
> In the past there was also a default node instead of latest kube context shown when user was not logged in or kubeconfig was missing. Was this changed?

yes we chenged that behavior in latest commits. we need to improve these 2 specific cases ( no kubeconfig/logged out )

**Comment by adietish** [comment]
with #630 merged things should behave correctly I think. Here's what I get if I trash my token and launch the plugin:
<img width="485" alt="image" src="https://github.com/redhat-developer/intellij-openshift-connector/assets/25126/b719eb51-16fe-4505-935f-2cb2e6f32e67">

**Comment by adietish** [comment]
@lgrossma not sure if I'm missing something, but there's no "Log out" command in the context menu for me.

**Comment by lgrossma** [comment]
@adietish You are right, it was there in the past and now it is not. However, I was reffering to a situation when user uses `oc logout` command from their terminal.

**Comment by adietish** [comment]
@lgrossma gotcha, trying it.

**Comment by adietish** [comment]
`oc logout` clears the token from the user in kube config.
